tests/test_interestingness.py

```diff
@@ -154,10 +154,8 @@ def test_diff_test_0() -> None:
             "diff_test",
             "--timeout",
             "99",
-            "-a",
-            "flags_one",
-            "-b",
-            "flags_two_a flags_two_b",
+            "-a 'flags_one'",
+            "-b 'flags_two_a flags_two_b'",
```

Comment:
What does this test do? These args look wrong.

Reply:
Fixed via [c6b41e7](https://github.com/MozillaSecurity/lithium/pull/92/commits/c6b41e7d573e772fcd933d66ba47c4f2b54793f6).